src/style.css

```diff
@@ -1 +1,12 @@
 /* styles go here */
+.input {
+  margin-bottom: 10px;
```

Comment:
Consider adding a more descriptive class name, such as 'form-input' instead of 'input', to make it clear that this class is related to the form.